book/08-customizing-git/sections/config.asc

```diff
@@ -1,28 +1,29 @@
 [[_git_config]]
-=== Git Configuration
+=== Kompigurasyon ng Git
 
 (((git commands, config)))
-As you read briefly in <<_getting_started#_getting_started>>, you can specify Git configuration settings with the `git config` command.
-One of the first things you did was set up your name and email address:
+Habang binabasa mo agad sa <<_getting_started#_getting_started>>, matutukoy mo ang Kompigurasyon ng Git sa `git config` na utos.
+Isa sa mga unang bagay na dapat mong gawin ay i-set up ang iyong pangalan at email address:
 
 [source,console]
 ----
 $ git config --global user.name "John Doe"
 $ git config --global user.email johndoe@example.com
 ----
 
-Now you'll learn a few of the more interesting options that you can set in this manner to customize your Git usage.
+Ngayon matututunan mo ang ilan sa mga mas interesanteng mga pagpipilian na maaari mong itakda sa ganitong paraan upang i-customize ang iyong paggamit sa Git.
 
-First, a quick review: Git uses a series of configuration files to determine non-default behavior that you may want.
-The first place Git looks for these values is in the system-wide `/etc/gitconfig` file, which contains settings that are applied to every user on the system and all of their repositories.
-If you pass the option `--system` to `git config`, it reads and writes from this file specifically.
+Una, isang mabilis na pagsusuri: Gumagamit ang Git ng isang serye ng mga kompigurasyon na file upang matukoy ang non-default behavior na maaaring gusto mo.
+Ang unang titingnan ng Git para sa mga halagang ito ay nasa buong sistema ng `/etc/gitconfig` na file, na naglalaman ng mga setting na inilapat sa bawat user sa system at lahat ng kanilang mga repositoryo.
```

Comment:
`ay nasa buong sistema ng `/etc/gitconfig` na file` to `ay ang buong sistema ng `/etc/gitconfig` na file`
`sa system at lahat ng kanilang mga repositoryo` to `sa sistem at lahat ng kanilang mga repositoryo`